db/model/bot.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or
+ * https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/**
+ * db/model/bot.js
+ *
+ * Bots are distributed webapps that will interact with our
+ * different routes. This object will be used to track
+ * and organize the bots that are associated with refocus.
```

Comment:
Can you add a little more information here about bot actions and data, how those fields are populated and used?